config/locales/en.yml

```diff
@@ -61,6 +61,9 @@ en:
     setup:
       intermediate_rent_product_name:
         blank: "Enter name of other intermediate rent product"
+      startdate:
+        later_than_14_days_after: "The tenancy start date must be not be later than 14 days from today's date"
+        before_scheme_end_date: "The start date of a supported housing tenancy must be before the supported housing scheme end date"
```

Comment:
@Dan13L3 I can't trigger these on the system locally so you'll have to review them here

Reply:
Why can't these be triggered locally?

Reply:
@Dan13L3 I've updated the description to show the screenshots I was able to grab when I triggered the errors locally